src/components/diagram-2a.js

```diff
@@ -9,7 +9,11 @@ class Diagram2a extends Component {
   }
 
   componentDidMount() {
-    observe(this.platform, 'cmp-diagram-chart__list--animated');
+    const diagrameObserver = () => {
```

Comment:
Super picky typo here 🙈 `diagrame` is `diagram` everywhere else in the codebase

Reply:
🤦‍♂